### PR TITLE
Add boilerplate Express app with docker environment

### PR DESCRIPTION
**Pull request by TravisWhitehead**

### CODE REVIEW

**Comment by nhcj** [comment]
Would you be fine with using PostgreSQL or MySQL instead?

**Comment by TravisWhitehead** [comment]
@jcorme Yeah IDC, I've just never stored files in SQL. Can't imagine it's difficult, though. Feel free to push a commit switching the DB stuff if you get to it before I do.

**Comment by nhcj** [comment]
@TravisWhitehead I'm thinking we can just store uploaded files in a GCS bucket. There's an official GCS Node library that looks pretty easy to use. I can handle that.

**Comment by TravisWhitehead** [comment]
@jcorme GCS as in Google Cloud Storage?

I switched the branch to Postgres.

File: db-init/00-create-user.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#
```
Comment (nhcj) [inline db-init/00-create-user.sh]: Remove this file.

File: lib/postgres.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const { Client } = require('pg');
+
+const db = new Client({
```
Comment (nhcj) [inline lib/postgres.js]: Use Pool instead of Client, should just be matter of replacing the word Client with Pool.

**Review by nhcj — CHANGES_REQUESTED** [review]

**Comment by nhcj** [comment]
Yes, Google Cloud Storage.

**Comment by TravisWhitehead** [comment]
@jcorme Done

**Review by nhcj — APPROVED** [review]

**Pull request merged**